scripts/start-baker.sh

```diff
@@ -66,6 +66,7 @@ usage() {
     echo "  --tezos-node <filepath>. Path for patched tezos-node executable"
     echo "  --tezos-baker <filepath>. Path for patched tezos-baker executable"
     echo "  --tezos-endorser <filepath>. Path for patched tezos-endorser executable"
+    echo "  --rpc-addr <rpc-addr>. Define RPC address of the baker node"
```

Comment:
Maybe it's better to make it optional and use `localhost:8732` as a default option